javascript/javascript1/week3/index.js

```diff
@@ -0,0 +1,232 @@
+//ITEM ARRAY REMOVAL
+
+// Remove the item in names array that is equal to nameToRemove
+
+const names = [
+  "Vartan",
+  "Robert",
+  "Mikkel",
+];
+const nameToRemove = "Robert";
+
+console.log(names);
+
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
+    break;
+  }
+}
+
+console.log(names);
+console.log("\n");
+
+///
+// WHEN WILL WE BE THERE??
+
+//Write a function where you speficy your speed in km/h and how far you have to go in km.
+//The function has to return the time it will take to arrive at your destination.
+//The time should be formatted like this: 3 hours and 34 minutes.
+
+function countTravelTime(travelInformation) {
+  let travelTimeInteger =
+    travelInformation.destinationDistance / travelInformation.speed;
+  let hours = Math.floor(travelTimeInteger);
```

Comment:
Variable naming conventions , hour/minute - is a general naming, be very specific.
ex: travelInHour etc...